eox_rates/apps_module/rating/models.py

```diff
@@ -0,0 +1,22 @@
+from model_utils.models import TimeStampedModel
+from django.conf import settings
+from django.db import models
+from django.db.models import CheckConstraint, Q, UniqueConstraint
+from django.contrib.auth.models import User
+from eox_rates.edxapp_wrapper.course_module import get_course_overview
+
+
+class EoxRates(models.Model):
+    """
+    Model to persist rating for a course.
+    """
+    rate = models.PositiveIntegerField(default=0, validators=[MinValueValidator(1), MaxValueValidator(5)])
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
This should be a ForeingKey too, and we should use UniqueConstraint for this.

Maybe something like this:
    class Meta:
        constraints = [
            models.UniqueConstraint(
                fields=["user", "rating"], name="unique_course_user_rating"
            )
        ]